src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -11,34 +11,35 @@
 import seedu.address.model.person.Name;
 import seedu.address.model.person.Person;
 import seedu.address.model.person.Phone;
+import seedu.address.model.person.Record;
 import seedu.address.model.person.RecordList;
 import seedu.address.model.tag.Tag;
 
 /**
  * Contains utility methods for populating {@code AddressBook} with sample data.
  */
 public class SampleDataUtil {
-    public static final RecordList EMPTY_RECORD_LIST = new RecordList();
+
     public static Person[] getSamplePersons() {
         return new Person[] {
             new Person(new Name("Alex Yeoh"), new Phone("87438807"), new Email("alexyeoh@example.com"),
                 new Address("Blk 30 Geylang Street 29, #06-40"),
-                getTagSet("friends"), EMPTY_RECORD_LIST),
+                getTagSet("friends"), getSampleRecordList()),
```

Comment:
Maybe this can be a Constant defined as a SAMPLE_RECORD_LIST if its being reused again.